tests/test_other.py

```diff
@@ -12168,9 +12168,9 @@ def test_rust_gxx_personality_v0(self):
         }
       }
     ''')
-    self.run_process([EMXX, '-c', 'main.cpp'])
+    self.run_process([EMXX, '-c', 'main.cpp', '-fexceptions'])
```

Comment:
Rust doesn't support Wasm EH yet right? It'd be good to add a TODO on top of this fine that we add `@with_both_eh_sjlj` later.

Reply:
Right. It would be great to work on adding wasm eh, though probably it's more of a job for you than for me. (I'm hoping to fix dynamic linking support.)

Reply:
Wasm EH uses a different name for this symbol doesn't it?  So I doubt this example will ever work (it will get undefined symbol error at link time).

Reply:
The test fails without the `-fexceptions` flag, so I think it is needed? Not sure about the defaults.

Reply:
Okay.